ex5_ch2.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Jun 13 10:08:53 2018
+
+@author: amyhaddad
+"""
+
+import math 
+
+#step1: write input statement asking for one number
+first_number = int(input("This is your first number: "))
+
+#step2: write input statement asking for second number
+second_number = int(input("This is your second number: "))
+
+#step2: create variable "sum" and write an equation that adds the two numbers together and print equation and solution 
```

Comment:
Nice comments. Right now, it's good to think about breaking your solutions into small chunks, as you're doing.